README.md

```diff
@@ -1,2 +1,70 @@
+- impacts the UX or operation of Istio substantially such that engineers using Istio will need retraining
+- users will notice and come to rely on
+
+It is unlikely an enhancement if it is:
+- fixing a flaky test
+- refactoring code
+- performance improvements, which are only visible to users as faster API operations, or faster control loops
+- adding error messages or events
+
+If you are not sure, ask someone in the working group where you initially circulated the idea.
+
+## When to Create a New Enhancement Issue
+
+Create an issue here once you:
+- have circulated your idea to see if there is interest
+   - through Community Meetings, working group meetings, discuss.istio.io lists, or an issue in github.com/istio/istio
+- (optionally) have done a prototype in your own fork
+- (optionally) have identified people who agree to work on the enhancement
+  - many enhancements will take several releases to progress through Alpha, Beta, and Stable stages
+  - you and your team should be prepared to work on the approx. 9mo - 1 year that it takes to progress to Stable status
+- (optionally) are ready to be the project-manager for the enhancement
+
+## Why are Enhancements Tracked
+
+Once users adopt an enhancement, they expect to use it for an extended period of time. Therefore, we hold new enhancements to a
+high standard of conceptual integrity and require consistency with other parts of the system, thorough testing, and complete
+documentation. As the project grows no single person can track whether all those requirements are met. The development
+of an enhancement often spans three stages: Alpha, Beta, and Stable; Enhancement Tracking Issues provide a
+checklist that allows for different approvers for different aspects, and ensures that nothing is forgotten across the
+development lifetime of an enhancement.
+
+## When to Comment on an Enhancement Issue
+
+Please comment on the enhancement issue to:
+- request a review or clarification on the process
+- update status of the enhancement effort
+- link to relevant issues in other repos
+
+Please do not comment on the enhancement issue to:
+- discuss a detail of the design, code or docs. Use a linked-to-issue or PR for that
+
+## Lifecycle of an Ehancement Issue
+
+- An enhancement issue must be presented to corresponding working group(s) meeting first, with a link to the RFC document.  If the enhancement spans more than one working groups, it must be presented to each of the working group to secure approvals.
+- Once the enhancement issue is approved by working group(s) and have engineer(s) who agrees to work on it, the enhancement issue must be approved by TOC.
+- The release managers can label the enhancement for one or more release(s).
+- PRs associated with the enhancement can be merged after approvers from more than 1 companies.
```

Comment:
Why is company relevant? Tim's keynote slide "project over company" seems relevant here

Reply:
oh wow I misread this. I thought you meant the KEP needed to be approved by more than one company, but you are suggesting each PR associated with that has to be approved by more than one company? That is a very strong claim, and makes Istio feel like less of an open project

Reply:
I prefer this corporate diversity requirement over no diversity requirement at all. The major key factor in any successful open source project is corporate diversity. The tech can be great, but if it is a single vendor project - it might as well be proprietary.

As for Tim's keynote, I'm not really clear how that is relevant to this community. I have seen many aspirational goals (e.g. OpenStack had project > people > company.  Tim seems to infer that CNCF adopts the model project > company. I think we can all agree we all have the project's best interests in mind when we take actions that affect the sustainability of the project.

Reply:
Isn't corporate diversity indirectly addressed through the working groups? ToC also has diversity and individual members can /hold if there are concerns that need to be addressed.

Reply:
Can we decouple this conversation from the rest of the IEP proposal? Lin put this on the TOC agenda which I think is a much more suitable audience. I would suggest we maintain status quo here and keep this PR to the core of adding an IEP process and make changes to our PR process in a separate discussion

Reply:
I suppose we could escalate to the TOC which might escalate to steering related to Istio's lack of corporate diversity.

I (may incorrectly assume) Lin is speaking on behalf of the TOC with this PR. This seems like a really good solution to bake corporate diversity into the project. I assume TOC will approve this README which will set the future direction of enhancements in Istio.

I don't see the value in an early escalation. The TOC can have a level-10 freak out when they review the readme if they find the corporate diversity requirement onerous...

Reply:
This is not early escalation, its already on the TOC agenda for tomorrow

Reply:
Having 2 companies reviewing each PR might probably exercise a lot of pressure on under-staffed companies / startups. I am not sure this will result into the expected diversity outcome.
Alternatively, maybe it could be required for a PR to be reviewed by at least 1 (one) Working Group leads and/or Maintainer for each Working Group related to that enhancement, as they would have been identified during the `## Is My Thing an Enhancement?` phase.

Reply:
folks, understood the concern, I agree projects > company if the project is owned by a foundation.  But istio is not... which is why diversity is critical for istio, especially requirements and user cases.    I reworded the requirements for approval for enhancement issue, which I definitely want corporate diversity.